test/videojs-http-streaming.test.js

```diff
@@ -58,6 +58,9 @@ QUnit.module('HLS', {
     this.old.NativeHlsSupport = videojs.Hls.supportsNativeHls;
     videojs.Hls.supportsNativeHls = false;
 
+    this.old.NativeDashSupport = videojs.Hls.supportsNativeDash;
+    videojs.Hls.supportsNativeDash = false;
```

Comment:
Edge supports native dash? or tries to, we need to stop that from happening

Reply:
yeah, edge supports some dash too. Probably will stop once chromiumEdge launches.